local.c

```diff
@@ -19,6 +19,10 @@
 #include "debug.h"
 #include "iio-private.h"
 #include "sort.h"
+#ifdef WITH_LOCAL_CONFIG
+#include "libini/ini.h"
```

Comment:
does clang-format have a preference for `#include "headers"` to be added after `#include <headers>`?
maybe it's a slightly better idea to move all this block after this;

but we can leave it as is for now;